### DISCUSSION TITLE
SQL Tables highlights

### DISCUSSION THREAD

**Discussion by TimBogevich** [terrastruct/d2#1575] [Q&A]
Hi. Thank you for your greate work first

I read the docs https://d2lang.com/tour/sql-tables and may see 
> If you hover over the primary key or foreign key with your cursor, you'll notice that the corresponding key is highlighted.
But it doesn't work.

But this behavior doesn't work. When I hover over any element nothing changes.

It also might be cool if you hover over a connection and it highlighted or changes it's color
It can solve other issues like when one element is covered or overlaid by another connection.

Thank you in advance

**Comment by alixander** [comment]
hey @TimBogevich sorry for the misleading doc there, I had actually just [removed this line yesterday](https://github.com/terrastruct/d2-docs/pull/183#issuecomment-1707415883) though it's not deployed yet.

Maybe will add as a feature later on, but for now it's unfortunately not a thing

**Reply by TimBogevich** [reply]
@alixander, thank you for your answer. 
Then I'll make this feature by myself via custom CSS


this is a simple workaround
.connection:hover {
  stroke: #eb4034 !important;
  stroke-width: 5 !important;
}
